examples/invoke-custom-data/README.md

```diff
@@ -82,7 +83,6 @@ Expected output from receiver:
 
 <!-- STEP
 expected_stdout_lines: 
-  - '✅  app stopped successfully: invoke-caller'
```

Comment:
is the caller app automatically stopped and if so why?

Reply:
The caller app is not a daemon, it exits automatically.